home-data/pom.xml

```diff
@@ -21,7 +21,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <scope>runtime</scope>
+            <version>42.2.16</version>
```

Comment:
I think it would be better to put this in a separate properties section.

Reply:
Thanks, made it